### PR TITLE
Classify fish using KNN data

### PR DESCRIPTION
**Pull request by madelynkasula**

### CODE REVIEW

File: src/activities/hoc2019/PondCreator.jsx [hunk 1]
```diff
@@ -8,6 +8,11 @@ const P5 = require('../../utils/loadP5');
 
 const FISH_COUNT = 9;
 
+export const Classifiers = Object.freeze({
```
Comment (bethanyaconnor) [inline src/activities/hoc2019/PondCreator.jsx]: really nit: isn't `Classes` a bit word for this than `Classifiers`? I _think_ classifiers are the things that actually do the classifying?

Comment (madelynkasula) [reply src/activities/hoc2019/PondCreator.jsx]: lol i'm not sure anymore but i like `Classes`!

Comment (bethanyaconnor) [reply src/activities/hoc2019/PondCreator.jsx]: Lol yeah I'm definitely not sure about any of the terminology...

Comment (madelynkasula) [reply src/activities/hoc2019/PondCreator.jsx]: maybe i'll do `ClassType` for now so that it isn't ambiguous with the loaded keyword class

**Pull request merged**